src/libsync/owncloudpropagator.h

```diff
@@ -98,7 +103,14 @@ class PropagatorJob : public QObject
     virtual qint64 committedDiskSpace() const { return 0; }
 
 public slots:
-    virtual void abort() {}
+    /*
+     * Asynchronous abort requires emit of abortFinished() signal,
+     * while synchronous is expected to abort immedietaly.
+    */
+    virtual void abort(PropagatorJob::AbortType abortType) {
```

Comment:
function requires "full definition" due to the fact that we call it using invokeMethod. Without it it thrown ```QMetaObject::invokeMethod: No such method OCC::PropagateDirectory::abort(PropagatorJob::AbortType)
Candidates are:
    abort(AbortType)```